packages/xo-server-sdn-controller/src/index.js

```diff
@@ -491,11 +491,10 @@ class SDNController extends EventEmitter {
             return
           }
 
-          let privateNetwork = this._privateNetworks[uuid]
-          if (privateNetwork === undefined) {
-            privateNetwork = new PrivateNetwork(this, uuid)
-            this._privateNetworks[uuid] = privateNetwork
-          }
+          const privateNetwork =
+            this.privateNetworks[uuid] !== undefined
+              ? this.privateNetworks[uuid]
+              : new PrivateNetwork(this, uuid)
```

Comment:
Why don't you update `this.privateNetworks` on creation anymore?

Reply:
I do it in the creator because it needs to be done for all private network creation.

Reply:
I see.